src/v3/src/components/Select/Select.tsx

```diff
@@ -69,93 +64,50 @@ const Select: UISchemaElementComponent<UISchemaElementComponentWithValidationPro
   };
 
   return (
-    <FormControl
-      disabled={loading}
-      error={hasErrors}
+    <NativeSelect
+      autoCompleteType={autocomplete}
+      errorMessage={errorMessage}
+      errorMessageList={errorMessageList}
+      hint={!required ? optionalLabel : undefined}
+      id={name}
+      inputRef={focusRef}
+      isDisabled={loading}
+      label={label}
+      onChange={(e: SelectChangeEvent<string>) => {
+        const selectTarget = (
+          e?.target as SelectChangeEvent['target'] & { value: string; name: string; }
+        );
+        handleChange?.(selectTarget.value);
+      }}
+      onBlur={(e: SelectChangeEvent<string>) => {
+        const selectTarget = (
+          e?.target as SelectChangeEvent['target'] & { value: string; name: string; }
+        );
+        handleBlur?.(selectTarget.value);
+      }}
+      testId={name}
+      value={value as string}
```

Comment:
This should be changed to 
```js
value={value ?? ''}
```

Otherwise Odyssey would set [`controlState`](https://github.com/okta/odyssey/blob/00daa33877428125405ce716101dd4c089081922/packages/odyssey-react-mui/src/NativeSelect.tsx#L162) to `UNCONTROLLED`
https://github.com/okta/odyssey/blob/00daa33877428125405ce716101dd4c089081922/packages/odyssey-react-mui/src/inputUtils.ts#L36-L37

This leads to a bug with rendering default value. 
Eg. for mock `authenticator-enroll-security-question` select should render [first option](https://github.com/okta/okta-signin-widget/blob/master/src/v3/src/transformer/layout/securityQuestion/securityQuestionEnroll.ts#L97) as default value:
https://github.com/okta/okta-signin-widget/blob/master/src/v3/test/e2e/screenshots/authenticator-enroll-security-question.png

On first render of `Select` component `value` prop is `undefined` (which leads to uncontrolled state detected by Odyssey, and it would not be changed to controlled), on second render `value` prop is `'disliked_food'`, but it will be rendered as "Select option"

Reply:
Ah, yeah, another good catch. I handled this uncontrolled state issue in a few of the other migrations but missed it here.